ocaml/lambda/lambda.ml

```diff
@@ -1430,9 +1430,12 @@ let primitive_result_layout (p : primitive) =
   | Paddfloat _ | Psubfloat _ | Pmulfloat _ | Pdivfloat _
   | Pbox_float _ -> layout_float
   | Punbox_float -> Punboxed_float
-  | Pccall _p ->
```

Comment:
Maybe we should add a case `Pccall _ when not !Clflags.native_code -> layout_any_value` ?